1. Use format() function with 145 and 'o':

In [1]:
result = format(145, 'o')
print("Result of format(145, 'o'):", result)

Result of format(145, 'o'): 221


Explanation:
The 'o' format converts the number into octal (base-8) representation.
So, 145 in decimal is 221 in octal.

2. Area of a circular pond with radius = 84 meters

    Formula: Area = π × r²

    π ≈ 22 / 7

In [2]:
pi = 22 / 7
radius = 84
area = pi * (radius ** 2)
print("Area of the pond:", area, "square meters")

Area of the pond: 22176.0 square meters


Bonus: Total water in liters (1.4 L per sq. meter)

In [3]:
water_per_sq_meter = 1.4
total_water = area * water_per_sq_meter
print("Total water in the pond:", total_water, "liters")

Total water in the pond: 31046.399999999998 liters


3. Speed when crossing 490 meters in 7 minutes

    Convert 7 minutes to seconds: 7 × 60 = 420 seconds

    Speed = Distance / Time

In [5]:
distance = 490
time_minutes = 7
time_seconds = time_minutes * 60

speed = distance / time_seconds
speed_no_decimal = int(speed)
print("Speed (in m/s) without decimal:", speed_no_decimal)

Speed (in m/s) without decimal: 1
